src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1702,50 +1717,56 @@ protected override void VisitArguments(ImmutableArray<BoundExpression> arguments
             ImmutableArray<int> argsToParamsOpt,
             bool expanded)
         {
-            VisitArguments(node, arguments, refKindsOpt, property is null ? default : property.Parameters, argsToParamsOpt, expanded);
+            MethodSymbol method = null;
+            VisitArguments(node, arguments, refKindsOpt, property is null ? default : property.Parameters, ref method, argsToParamsOpt, expanded);
         }
 
         private void VisitArguments(
             BoundExpression node,
             ImmutableArray<BoundExpression> arguments,
             ImmutableArray<RefKind> refKindsOpt,
             ImmutableArray<ParameterSymbol> parameters,
+            ref MethodSymbol method,
```

Comment:
>ref MethodSymbol method, [](start = 12, length = 24)

It might be clearer to change the method to return possibly adjusted method symbol instead of passing it by reference.

Reply:
Note: I made that change after your last review. But since then I squashed and pulled new bits from the feature branch to resolve conflicts.